platform-core-module/src/main/java/com/softicar/platform/core/module/daemon/IDaemon.java

```diff
@@ -13,6 +13,17 @@
 @FunctionalInterface
 public interface IDaemon {
 
+	/**
+	 * Executes setup operations to perform before entering the daemon's main
+	 * loop.
```

Comment:
We should be more specific. Is this called only once, or before every `runIteration()`...same goes for the `cleanup()` method.

Reply:
done